### DISCUSSION TITLE
Make keyboard-shortcuts optional

### DISCUSSION THREAD

**Discussion by hellodword** [community/community#23163] [New to GitHub]
<p>I believe <code>keyboard-shortcuts</code> is a wonderful feature for keyboard lovers, but I think it’s better if this feature can be optional for users.</p>
<p>I’m not a keyboard ninja, so for me, <code>keyboard-shortcuts</code> on github often cause accidental forks (touch <code>e</code> when viewing source code), sometimes awkward😂.</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/hellodword">@hellodword</a> , we appreciate the suggestion.</p>
<p>There presently is not a way to disable shortcuts for the site. We recommend getting to know what shortcuts are available and what they do. They can improve your workflow and help you to avoid accidental actions.</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="15" height="15">
      <a href="https://docs.github.com/en/github/getting-started-with-github/keyboard-shortcuts" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/getting-started-with-github/keyboard-shortcuts" target="_blank" rel="noopener">Keyboard shortcuts - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/getting-started-with-github/keyboard-shortcuts" target="_blank" rel="noopener">//docs.github.com/en/github/getting-started-with-github/keyboard-shortcuts</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>We’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.<br>
You can do so by <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request.</p>
<p>You can also keep an eye on our <a href="https://github.blog/changelog/">Changelog</a> as well as the <a href="https://github.com/github/roadmap">GitHub public roadmap</a> to see features we are working on.</p>
